client/head.jade

```diff
@@ -0,0 +1,7 @@
+head
+  meta(charset="utf-8")
+  title NIA Monitoring
```

Comment:
It should be NIA Monitor. "NIA Monitoring" has been used in a few places already but I'd like to keep it from propagating further so it's easier to fix.